### ISSUE TITLE
Create rule S6841 (`jsx-a11y/tabindex-no-positive`): `tabIndex` values should be non-positive

### ISSUE THREAD

**Issue by ilia-kebets-sonarsource** [SonarSource/SonarJS#4281]
[plugin rule](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/tabindex-no-positive.md#jsx-a11ytabindex-no-positive)
